io/disk/ltp_fs.py.data/ltp_fs.yaml

```diff
@@ -0,0 +1,6 @@
+general:
+    disk: ''
+    script: 'runltp'
+    args: '-s fs_di,-s fs_inod'
```

Comment:
Sure. I have accounted for ext2, ext3 and ext4 filesystems. If the user wants more, he/she can add